cruise.umple/src/Main_Code.ump

```diff
@@ -569,6 +569,7 @@ class UmpleConsoleMain
     suboptionsAvail += "(With GvClassDiagram or GvClassTraitDiagram or GvEntityRelationshipDiagram: hideattributes, showmethods)";
     suboptionsAvail += " (With all Gv: gvseparator=n.n where n.n is greater than 1.0 to separate nodes more than default, or less than one to make the diagram tighter)";
     suboptionsAvail += " (With all Requirements: reqSortID to sort requirements by their ID, reqSortStat to sort requirements by their Statement)";
+    suboptionsAvail += " (With all Requirements: reqSortImpl to filter requirements that are implemented, reqSortNotImpl to to filter requirements that are Not implemented)";
```

Comment:
This is not about 'sorting'. Also it is best to just have one with the default beiing to show them all. So Please change so there is reqHideNotImpl as the only one, with the other being the default.